AdvancedApplications/JDBCApplication/README.md

```diff
@@ -1,6 +1,15 @@
-# Instructions on How to Run the IBM Streams and Remote Applications For IBM Db2 Event Store
+# Using the sample JDBC Remote Application
 
-## Using the sample JDBC Remote Application
+The simplest option is to use the Docker container provided in this repository that is configured to run the application.  The JCBC application creates the table ADMIN.IOT_TEMP, inserts a number of rows from a csv file and queries the first 10 rows. 
+
+## Container option
+
+1. In runExampleJDBCApp change the client jar defined with ESLIB to the directory where the client Spark jar file, for example ibm-db2-eventstore-client-spark-2.2.1-2.0.0.jar, is located. The Spark file was obtained from Maven earlier [here](https://mvnrepository.com/artifact/com.ibm.event/ibm-db2-eventstore-client-spark-2.2.1)). This JAR includes the Db2 Standard JDBC client. Alternatively, you can download the Db2 JDBC client driver from the IBM Data Server Client Packages page under the IBM Support [website](https://www-01.ibm.com/support/docview.wss?uid=swg21385217).
```

Comment:
I think there will be some confusion here e.g.

We point to a spark jar jdk yet we're saying it can be replaced with a standard out of the box jdbc client driver.

That is accurate mind you, but I think we should consider 1)  Using a standard IBM JDBC client driver directly for clarity or 2)  Explaining that for convivence we just use our own spark client library which contains an emended jdbc driver (i.e. db2jcc4.jar) as said spark client library is also used for our Scala/Python/Java examples.

Reply:
I've reached out to Chen to get some background on why we suggest 2 paths to get the client,

Reply:
yes, I agree, it would be better to recommend the standard IBM JDBC client, as with this option not only it is a more general option, but also more flexible, as the use can use any version of this. 

Like Hamdi says, the alternative option is to use our own spark client library that has a specific IBM JDBC client **embedded** within it, so provides the same functionality, but restricted to a single version of the IBM JDBC client.

Reply:
removed the IBM Data Server Client Packages reference so the person would just use the Db2 Standard JDBC client